src/main/java/org/wso2/lsp4intellij/utils/GUIUtils.java

```diff
@@ -49,7 +67,33 @@ public static Hint createAndShowEditorHint(Editor editor, String string, Point p
      * @return The hint
      */
     public static Hint createAndShowEditorHint(Editor editor, String string, Point point, short constraint, int flags) {
-        LightweightHint hint = new LightweightHint(new JLabel(string));
+        JTextPane textPane = new JTextPane();
+        textPane.setEditorKit(new HTMLEditorKit());
+        textPane.setText(string);
+        textPane.setEditable(false);
+        textPane.addHyperlinkListener(new HyperlinkListener() {
+            @Override
+            public void hyperlinkUpdate(HyperlinkEvent e) {
+                if ((e.getEventType() == HyperlinkEvent.EventType.ACTIVATED)
+                        && !Strings.isNullOrEmpty(e.getDescription())) {
+                    try {
+                        if ("http".equals(e.getURL().getProtocol())) {
+                            BrowserLauncher.getInstance().browse(e.getURL().toURI());
+                        } else {
+                            final Project project = editor.getProject();
+                            VirtualFile file = VfsUtil.findFileByURL(new URL(VfsUtilCore.fixURLforIDEA(e.getURL().toString())));
+                            final OpenFileDescriptor descriptor = new OpenFileDescriptor(project, file);
+                            writeAction(() -> {
+                                FileEditorManager.getInstance(project).openTextEditor(descriptor, true);
+                            });
+                        }
+                    } catch (MalformedURLException | URISyntaxException ex) {
+                        LOGGER.error("Error opening doc link", ex);
```

Comment:
Shall we prompt a message to the user instead of showing logging this as an error (which will be shown as an fatal error in the IDE)? WDYT?

Reply:
Sure may be we can log this as a debug message and show a notification which will be shown in the event log. WDYT ?

Reply:
I'm afraid that the event log will be noisy if we continue prompting these kind of exceptions since it's shared by all the tools for prompting critical events :/  How about the IntelliJ Messages API? 

```
Messages.showErrorDialog(message, title);
```

Reply:
I meant to just show a info event in the event log. I don't see that we will get lot of such errors ?
Of cause we can use the Message API but then user will need to close the popup right ? Will that be better from UX point of view ?.

Reply:
@gayanper @NipunaRanasinghe my suggestion is also to move ahead with the Messages API. If we are going to add this into event log; we need to treat & add all error messages in other places in same level and I am afraid that will flood the events log. IMO since this action is user-initiated (clicking the hyperlink); showing error message in a failure won't harm the UX. WDYT?